### PR TITLE
fix: Autoteleport after rfc - lowering deposits and fees

### LINKED ISSUE #10151: Autoteleport after rfc - lowering deposits and fees
How is it with Autoteleport currently? Deposits on AHP were lowered. Can we have a look at Autoteleport again and maybe lower the requirements? 

image
image
image

looks like it works, and it will leave the account with 0.1 DOT

the existential deposit is way lower now

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

`ahp` ed from `0.1` to `0.01`

- [x] Closes #10151

## Needs QA check

- @kodadot/qa-guild please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [ ] My fix has changed **something** on UI;

### CODE REVIEW

**Review by vikiival — APPROVED** [review]

File: libs/static/src/chains.ts [hunk 1]
```diff
@@ -109,7 +109,7 @@ export const existentialDeposit: Record<Prefix, number> = {
   rmrk: 333333333,
   ahk: 333333333,
   dot: 10000000000,
-  ahp: 1000000000,
+  ahp: 100000000,
```
Comment (preschian) [inline libs/static/src/chains.ts:112]: Can we use a numeric separator here? `100_000_000`

**Review by preschian — APPROVED** [review]
no changes on `ahk`?

**Review by Jarsen136 — APPROVED** [review]

File: libs/static/src/chains.ts (hunk 1 above)
Comment (vikiival) [reply libs/static/src/chains.ts:112]:
```suggestion
  ahp: 1e8,
```

Comment (hassnian) [reply libs/static/src/chains.ts]: updated 85e5b10

**Review by vikiival — APPROVED** [review]

**Pull request merged**